File: Baekjoon/DP/1003.py
# https://www.acmicpc.net/problem/1003

import sys
input = sys.stdin.readline

for _ in range(int(input())):
    t = int(input())
    dp = [[0, 0] for _ in range(t + 1)]
    dp[0] = [1, 0]
    if t > 0:
        dp[1] = [0, 1]
        for i in range(2, t + 1):
            dp[i][0] = dp[i-1][0] + dp[i-2][0]
            dp[i][1] = dp[i-1][1] + dp[i-2][1]
    print(dp[t][0], dp[t][1])

File: Baekjoon/DP/1003.cpp
// https://www.acmicpc.net/problem/1003

#include <iostream>
#include <vector>
using namespace std;

int main() {
    int t;
    cin >> t;

    vector<pair<int, int>> dp(41);
    dp[0] = {1, 0};
    dp[1] = {0, 1};
    for (int i = 2; i <= 40; i++) {
        dp[i].first = dp[i - 1].first + dp[i - 2].first;
        dp[i].second = dp[i - 1].second + dp[i - 2].second;
    }

    while (t--) {
        int n;
        cin >> n;
        cout << dp[n].first << ' ' << dp[n].second << '\n';
    }

    return 0;
}